pkg/api/v1/conversion.go

```diff
@@ -373,63 +372,14 @@ func Convert_api_PodStatusResult_To_v1_PodStatusResult(in *api.PodStatusResult,
 		return err
 	}
 
```

Comment:
I think Convert_api_PodTemplateSpec_To_v1_PodTemplateSpec and Convert_v1_PodTemplateSpec_To_api_PodTemplateSpec can also be removed